lib/bundle/brew_services.rb

```diff
@@ -1,5 +1,11 @@
 module Bundle
   class BrewServices
+    def self.stop(name)
+      started = !`brew services list | grep "#{name}" | grep -q "started"`.chomp.empty?
```

Comment:
Oh, a final improvement. It may be a better idea to use `split/include?` instead of shelling out using `grep`. This is to avoid shell injection.